worlds/smz3/__init__.py

```diff
@@ -377,6 +378,24 @@ def get_filler_item_name(self) -> str:
     def write_spoiler(self, spoiler_handle: TextIO):
             self.world.spoiler.unreachables.update(self.unreachable)
 
+    def JunkFillGT(self):
+        locationNamesGT = [loc.Name for loc in self.smz3World.GetRegion("Ganon's Tower").Locations]
```

Comment:
this should be a set for faster lookups, it could also be computed once and saved somewhere.